styles/less/fix.less

```diff
@@ -1,3 +1,4 @@
+// Fix specific style added by some add-ons
```

Comment:
unused file?

Reply:
This file will contain all style used to force the correction of the aspect for the elements added by some plugins, when we can't diretely have the access to the CSS sources file of it. (in this case here, it is to force the correct position of the chat bubble).